docs/make.bat

```diff
@@ -5,7 +5,7 @@ REM Command file for Sphinx documentation
 if "%SPHINXBUILD%" == "" (
 	set SPHINXBUILD=sphinx-build
 )
-set BUILDDIR=_build
+set BUILDDIR=../../oceanparcels_website/gh-pages

```

Comment:
This assumes something about the structure of the directories for the repos. Why did this need to change?

Reply:
good point, the 'make html' command would not output the html files in the oceanparels_website folder for me otherwise. 

In 'Makefile' the builddir was already implemented in a similar way in line 8: 
`BUILDDIR      = ../../oceanparcels_website/gh-pages`
This suggests that the structure of the directories is the same, but somehow my (windows?) device gets the directory from 'make.bat' whereas on other devices the builddir is read in 'Makefile'?

Reply:
OK, let's leave it then as is